### ISSUE TITLE
Low accuracy in recognition models

### ISSUE THREAD

**Issue by sadimoodi** [littletomatodonkey/insight-face-paddle#20]
Hello,
I am facing very low accuracy in both recognition models (ArcFace and MobileFace), no matter how i play with the input params (such as Rec_thresh), i am not getting good results at all.
what could be wrong?

**Comment by littletomatodonkey** [comment]
You can try only-det mode to find out whether the detector is the bottleneck, and the index is built by the recognition model on `friends`, you can try to see whether the index is right.

**Comment by sadimoodi** [comment]
I have tried setting the args.det = True and i can confirm that it FAILS to detect big faces, such as:
![Mike](https://user-images.githubusercontent.com/32739021/142376238-601e926d-830a-4eb1-9ae7-56692f832e6b.jpg)
however, i don't face an issue in medium size faces.
BIG problem: recognition is very poor even on medium size / very clear faces
something seems to be fundamentally wrong with this library as i have tried changing all params and nothing worked

**Comment by littletomatodonkey** [comment]
yes, the current small detection model does not perform well on image with medium or large face, you can try replace the model with other models.
